Database/Constraint/Checks/Podium Checks.sql

```diff
@@ -25,13 +25,21 @@ GO
 /* Testdata */
```

Comment:
Style ontbreekt. Er staat bovenaan niet wie deze constraint heeft gemaakt, op welke dag en datum en wat de naam van de business rule is..

Even de Style aanvullen.